pkg/volume/gce_pd/attacher.go

```diff
@@ -165,6 +165,9 @@ func (attacher *gcePersistentDiskAttacher) MountDevice(spec *volume.Spec, device
 	options := []string{}
 	if readOnly {
 		options = append(options, "ro")
+	} else {
+		// as per https://cloud.google.com/compute/docs/disks/add-persistent-disk#formatting
+		options = append(options, "discard")
```

Comment:
According to the same documentation don't we also want to do this for mkfs?
Do we need to add it also in [mount_linx.go]?(https://github.com/kubernetes/kubernetes/blob/master/pkg/util/mount/mount_linux.go#L282)?